test/IDE/newtype.swift

```diff
@@ -92,7 +92,7 @@
 // PRINT-NEXT:  extension CFNewType {
 // PRINT-NEXT:    static let MyCFNewTypeValue: CFNewType
 // PRINT-NEXT:    static let MyCFNewTypeValueUnauditedButConst: CFNewType
-// PRINT-NEXT:    static var MyCFNewTypeValueUnaudited: Unmanaged<CFString>
+// PRINT-NEXT:    static var MyCFNewTypeValueUnaudited: Unmanaged<CFNewType>
```

Comment:
Why the change here?

Reply:
It's a good question, then answer is I'm not quite sure. I think it's OK because `CFNewType` is a clang-side typedef to `CFString` (and it's arguably more correct because it's inline with what `MyCFNewTypeValueUnauditedButConst` does and it's the type clang uses).

Reply:
The clang importer may be relying on the getMembers call here to fault something in. You should check `CFPointeeInfo::classifyTypedef` between the two compilers.